Overview.src.html

```diff
@@ -4734,6 +4746,10 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  <var>request</var>'s
  <span title=concept-request-integrity-metadata>integrity metadata</span> to it.
 
+ <li><p>If <var>init</var>'s <code title>keepAlive</code> member is present, set
```

Comment:
I can change that, but (fyi) it's inconsistent with surrounding language.. e.g.

>  <li><p>If <var>init</var>'s <code title>integrity</code> member is present, set
 <var>request</var>'s <span title=concept-request-integrity-metadata>integrity metadata</span> to it.

I'll keep as is for now, let me know if you want me to change it.